tensorboard/plugins/audio/summary.py

```diff
@@ -19,6 +19,8 @@
 the tensor is a pair `[encoded_audio, label]`, where `encoded_audio` is
 a binary string whose encoding is specified in the summary metadata, and
 `label` is a UTF-8 encoded Markdown string describing the audio clip.
+
+NOTE: This module is in beta, and its API is subject to change.
```

Comment:
Feel free to append, "but the data it stores to disk will be supported forever."